JS/home.js

```diff
@@ -0,0 +1,108 @@
+      navContainer.innerHTML = `
+          <nav class="navbar navbar-expand-lg bg-light">
+              <div class="container-fluid" style="display: flex; justify-content: space-between">
+                  <a class="navbar-brand" href="#">LMS</a>
+                  <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav"
+                      aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                      <span class="navbar-toggler-icon"></span>
+                  </button>
+                  <div class="collapse navbar-collapse" id="navbarNav">
+                      <ul class="navbar-nav">
+                          <li class="nav-item">
+                              <a class="nav-link active me-2" aria-current="page" href="../Html-files/home.html">Home</a>
+                          </li>
+                          <li class="nav-item">
+                          <a class="nav-link active me-2" aria-current="page"
+                              href="../Html-files/techer.html">Teachers</a>
+                              <li class="nav-item">
+                              <a class="nav-link active me-2" aria-current="page"
+                                  href="../Html-files/students.html">Students</a>
+                      </li>
+                          <li class="nav-item">
+                              <a class="nav-link active me-2" aria-current="page" href="../Html-files/Class.html">Classes</a>
+                          </li>
+                          
+                          </div>
+                          <div class="mx-2">
+                              <i class="far fa-user-circle fa-2x"></i>
+                          </div>
+                      </div>
+                      </ul>
+                  </div>
+              </div>
+          </nav>
+      `;
+    }
+  
+    // Update the welcome message dynamically
+    const welcomeMessage = document.getElementById("welcomeMessage");
+    welcomeMessage.textContent = "Welcome Mirjam";
+  
+    function addMainContent() {
+      const mainContent = document.getElementById("mainContent");
+      mainContent.innerHTML = `
+          <section id="cards">
+          <article class="box-container" id="studentsCard" style="height: 200px; width: 300px">
+                  <h2>22</h2>
+                  <p>number of students</p>
+              </article>
+              <article class="box-container" id="teachersCard"style="height: 200px; width: 300px">
+                  <h2>3</h2>
+                  <p>number of teachers</p>
+              </article>
+              <article class="box-container" id="classesCard"style="height: 200px; width: 300px">
+                  <h2>5</h2>
+                  <p>number of classes</p>
+              </article>
+          </section>
+      `;
+    }
+    function addHeading() {
+      const heading = document.createElement("h2");
```

Comment:
Do not use DOM manipulation APIs, only innerHTML

Reply:
Ok sir